graphbolt/src/index_select.cc

```diff
@@ -21,6 +22,25 @@ torch::Tensor IndexSelect(torch::Tensor input, torch::Tensor index) {
   return input.index({index.to(torch::kLong)});
 }
 
+torch::Tensor DiskIndexSelect(
+    std::string path, torch::Tensor index, torch::ScalarType dtype) {
+#ifdef __linux__
+  storage::OnDiskNpyArray arr(path);
```

Comment:
What is the overhead of opening the file everytime DiskIndexSelect is called?

Reply:
We could keep an open handle instead of working with paths.

Reply:
Good idea.